### PR TITLE
[CONFORMANCE] Add ReadMe file for new SubgraphsDumper tool

### PR DESCRIPTION
**Pull request by iefode**
### Details:
 - *[Link to ReadMe](https://github.com/iefode/openvino/blob/read_me/src/tests/functional/plugin/conformance/subgraphs_dumper_new/README.md)*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/tests/functional/plugin/conformance/subgraphs_dumper_new/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Validation
+The tool is validated by combination of Unit and Functional tests using based OV approach. To run tests using the following commands:
+```
+cmake -DENABLE_FUNCTIONAL_TESTS=ON .
+make subgraphsDumperTests
+./subgraphsDumperTests --gtest_filter=*
+```
+
+## Architecture diagram
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/subgraphs_dumper_new/README.md:48]:
```suggestion
## Architecture Diagram
```

**Comment by iefode** [comment]
@tsavina Thanks a lot for review! I have applied your comments. Could you please take the review a look one time?

**Comment by iefode** [comment]
@tsavina Could you please review the changes?

**Comment by iefode** [comment]
@andrei-kochin Please approve. I will merge myself

**Review by andrei-kochin — APPROVED** [review]

**Pull request merged**